src/devices/bresser_7in1.c

```diff
@@ -11,11 +11,16 @@
 
 #include "decoder.h"
 
-#define SENSOR_TYPE_WEATHER 1
-#define SENSOR_TYPE_AIR_PM  8
+#define SENSOR_TYPE_WEATHER   1
+#define SENSOR_TYPE_AIR_PM    8
+#define SENSOR_TYPE_CO2      10
+#define SENSOR_TYPE_HCHO_VOC 11
 
 /**
-Decoder for Bresser Weather Center 7-in-1 and Air Quality PM2.5 / PM10, outdoor sensors.
+Decoder for Bresser Weather Center 7-in-1 and Air quality sensors.
+Air Quality PM2.5/PM10 PN 7009970
```

Comment:
This is markdown, for nice display the three device lines should start with a dash `- Air…`